src/nmea/nmea.c

```diff
@@ -210,7 +212,7 @@ nmea_free(nmea_s *data)
 }
 
 nmea_s *
-nmea_parse(char *sentence, size_t length, int check_checksum)
+nmea_parse(char *sentence, size_t length, int check_checksum, int check_escape_chars)
```

Comment:
This change would mean that everyone who is using `libnmea` would have to modify their code to work with the new function prototype.

I wonder if you have considered some alternative approaches, which wouldn't be a breaking change for other users of the library?

Reply:
That's what I thought, this is a change that fit with my use.

Simple solutions:
1. Simply disable/enable this check at library compile time via macro directive.

1. Or create two separate function and then enable/disable before importing the library:
    
    `nmea.h`
    ```  c
    #ifndef NMEA_CHECK_ESCAPE_CAHRS
    nmea_parse(char *sentence, size_t length, int check_checksum)
    #else
    nmea_parse(char *sentence, size_t length, int check_checksum, int check_escape_chars)
    #endif
    ```
    `main.c`
    ``` c
    #define NMEA_CHECK_ESCAPE_CHARS
    #include <nmea.h>
    // This enable the prototype with the optional escape chars validation
    ```
    But i don't really like this solution, because that means write two "identical" functions.

Reply:
I don't have access to the official NMEA standard, but all the references I could find specify the sentences in the following way:
```
$ttsss,d1,d2,....<CR><LF>
```
So it seems like CR and LF _are_ part of standard NMEA messages. Therefore i think the correct behavior for the library would be to expect them to be part of the message.

If you are loading data from a file which only has <LF> characters and not <CR>, you can easily modify each line after reading it from the file, replacing <LF> with <CR><LF>.

Reply:
Yes the NMEA standard should follow the example, so this PR is useless, I'm closing it